packages/driver/src/cy/commands/asserting.ts

```diff
@@ -7,6 +7,30 @@ import $errUtils from '../../cypress/error_utils'
 const reExistence = /exist/
 const reHaveLength = /length/
 
+const onBeforeLog = (log, command, commandLogId) => {
+  log.set('commandLogId', commandLogId)
+
+  if (command) {
```

Comment:
When would command be undefined here?

Reply:
It wouldn't be here. Remnant of when this was in log.ts, where there might not be a current command. Will remove the check.